includes/tax-functions.php

```diff
@@ -442,3 +410,60 @@ function edd_download_is_tax_exclusive( $download_id = 0 ) {
 
 	return (Bool) apply_filters( 'edd_download_is_tax_exclusive', $ret, $download_id );
 }
+
+/**
+ * Gets the tax rate object from the database for a given country / region.
+ * Used in `edd_get_tax_rate`, `edd_build_order`, `edd_add_manual_order`.
+ *
```

Comment:
Suggest adding a blurb about the internal fallback logic so people using this function know what to expect.

I.E. Does a Region override a Country? (which is being discussed elsewhere)

Reply:
> Maybe should set $args = array() in the function parameters

I like everything you said, @JJJ , however, I'd personally keep the parameter as `$args` just because having a parameter is _required_. I'd only do `$args = array()` if the args were optional.